.github/workflows/ci.yml

```diff
@@ -342,44 +342,104 @@ jobs:
           name: OpenRCT2-${{ needs.build_variables.outputs.name }}-Windows-mingw-${{ matrix.platform }}
           path: bin/openrct2.exe
           if-no-files-found: error
-  macos-cmake:
-    name: macOS (${{ matrix.arch }}) using CMake
+  macos-x64-cmake:
```

Comment:
Why were these split into two jobs, rather than continuing to use the `matrix.arch` parameter to _make_ two jobs?

Reply:
> Why were these split into two jobs, rather than continuing to use the matrix.arch parameter to make two jobs?

there is some issue finding the native openssl, https://github.com/OpenRCT2/OpenRCT2/actions/runs/12591400084/job/35094478800

Reply:
I'm still not overly happy with the duplication, though.

@janisozaur Do you have any ideas how this can be improved?